common/src/main/java/com/wynntils/wrappedscreens/trademarket/TradeMarketSearchResultHolder.java

```diff
@@ -0,0 +1,369 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wrappedscreens.trademarket;
```

Comment:
Is a wrappedscreen a new kind of top level consumer, like functions, features and screens?

Reply:
Do you want them to be subclassed to screens? Otherwise, that should be their location

Reply:
Yes, I rather want them under `screens`. At least for the moment; I might have to think a bit about this when we get more of them.